agent/api/eni/eniattachment.go

```diff
@@ -23,8 +23,15 @@ import (
 	"github.com/pkg/errors"
 )
 
+const (
+	ENIAttachmentTypeENI      = "eni"
```

Comment:
lint will tell you that you need to document these global scoped const's

Reply:
there is currently a large amount of output from golint. fixed the ones brought by my changes including this